### PR TITLE
add support for link preload hrefs

### PR DESCRIPTION
**Pull request by akrawchyk**
This PR adds support for `preload` links as specified here https://w3c.github.io/preload/.

### CODE REVIEW

File: test.html [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        inrfood : {
+          title : "INRFOOD",
+          location : "Raleigh, NC",
+          description : "The prize money will help expand our platform with features we've been testing, including food suggestions and an API for other apps/tools to connect to.",
+          body : "INRFOOD is a personalized nutrition platform that, upon scanning the bar code of over 300,000+ products, instantly analyzes the nutrition facts and more importantly the ingredients. It highlights any ingredients that may pose a medical, allergic or cultural concern specifically to the user.",
+          url : "<iframe id=\"player\" type=\"text/html\" src=\"http://www.youtube.com/embed/75jP-rjqsMI\" height=\"300\" width=\"100%\" frameborder=\"0\"></iframe>",
+          like : ""
+
+        }
+    }</script><script src="//platform.twitter.com/oct.js"></script><script>twttr.conversion.trackPid('l4mg5');</script><noscript><img height="1" width="1" style="display:none;" alt="" src="https://analytics.twitter.com/i/adsct?txn_id=l4mg5&p_id=Twitter" /> <img height="1" width="1" style="display:none;" alt="" src="//t.co/i/adsct?txn_id=l4mg5&p_id=Twitter" /></noscript></body></html>
```
Comment (addyosmani) [inline test.html]: I can't help but feel like this is a little OTT for a test page. Are we using this here for a specific reason? If not, would it be possible to simplify it down into something that doesn't rely on as much third-party content and scripts?

**Comment by addyosmani** [comment]
PR LGTM minus the test.html file used. @akrawchyk could you comment on that and we can work on getting this landed soon and chase up the PR in critical right after?

File: test.html (hunk 1 above)
Comment (akrawchyk) [inline test.html]: Whoops, I think that is a copy pasta error, my mistake! I'll replace it with a smaller test file.

File: test/sample/index.html [hunk 2]
```diff
@@ -8,6 +8,7 @@
         <!-- Place favicon.ico and apple-touch-icon.png in the root directory -->
         <link rel="stylesheet" href="bower_components/bootstrap/dist/css/bootstrap.css" />
         <link rel="stylesheet" href="styles/main.css">
+        <link rel="preload" href="styles/preload.css" as="style">
```
Comment (akrawchyk) [inline test/sample/index.html]: @addyosmani in reference to https://github.com/addyosmani/oust/pull/19/files#r67332062, this is the only adjustment I made for the test. The `test.html` file was for my use case and wasn't meant to be committed.

**Pull request merged**